packages/commands/src/handlers/new.ts

```diff
@@ -133,17 +136,17 @@ const handleTSConversion = async (tempDir: string, projectName: string) => {
 	});
 };
 
-const handleNewStartProject = async (projectName: string, variation?: AllSupported) => {
-	const template = await cancelable(
+const handleNewStartProject = async (projectName: string, template?: Template, extension?: Extension) => {
```

Comment:
Passing `extension` doesn't make sense here I don't think.
Originally, `localSupported` represented all the possible vanilla SolidJS templates.

Reply:
Sorry, I'm not sure what change you're looking for. Should I _not_ pass in `extension`? If so, that has upstream and downstream effects that (IMO) are out of scope. Right now this PR fixes two things: the title, and renaming. I'm trying to keep this PR small in the interest of a smaller diff. If you're saying that extension should be dropped in favor of `template`, so that `js`, `ts`, `bare`, `basic`, `hackernews`, etc. are listed together, `js` and `ts` are [not examples in the solid-start repo](https://github.com/solidjs/solid-start/tree/main/examples), so I don't think that'd work without more coordination.

Reply:
@Tommypop2 bump!